src/core/animation-loop.js

```diff
@@ -7,11 +7,13 @@ import assert from 'assert';
 
 // Node.js polyfills for requestAnimationFrame and cancelAnimationFrame
 export function requestAnimationFrame(callback) {
-  return isBrowser ? window.requestAnimationFrame(callback) : setTimeout(callback, 1000 / 60);
+  return isBrowser && self.requestAnimationFrame ?
```

Comment:
Q: These are being updated to support offscreen, but looks like workers/offscreen mode don't use requireAnimationFrame below? 
 Instead chaining promises?